frontend/src/components/CostTableMenu.vue

```diff
@@ -190,7 +165,7 @@ const handleUpdateChargeType = (val: ChargeType) => {
   emit("update-charge-type", val);
 };
 
-const handleUpdateEngineType = (val: EngineType) => {
+const handleUpdateEngineType = (val: any) => {
```

Comment:
why use any here?

Reply:
Because this function is used by Naive-UI's during updating, the assertion here is `MaybeArray<(value: (string | number)[]) => void> | undefined`. Any other value will cause build error.